ih-core/lib/steps/shell/custom/00_env.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# This file defines the user-specific environment variables
+# which are expected by other engineering scripts,
+# as well as any additional things you want to add.
+
+# This file will not be updated when you update the ih-core brew formula.
+
+# Directory where you want to clone Legacy Grand Rounds repos,
+# which are currently located in the ConsultingMD org.
+export GR_HOME="$HOME/src/github.com/ConsultingMD"
+
+# Directory where you want to clone Legacy Doctor on Demand repos,
+# which are currently located in the doctorondemand org.
+export DOD_HOME="$HOME/src/github.com/doctorondemand"
+
+# Your GitHub username
+export GITHUB_USER=
+
+# Your email address
+export EMAIL_ADDRESS=
+
+# Your 3 character initials
+export INITIALS=
+
+# Your Legacy Grand Rounds username, probably firstname.lastname
+export GR_USERNAME=
+
+# The username you have in JIRA.
+# As of 11/2021 :
+# if your email address is @grandrounds.com or @includedhealth.com,
+# your JIRA username is probably "$GR_USERNAME@grandrounds.com"
```

Comment:
Don't we all have `@includedhealth` addresses now? Which would make this not accurate?

Reply:
That doesn't seem to have propagated to Atlassian yet. If I try to log in with steve.ruble@includedhealth.com it asks me for a password; steve.ruble@grandrounds.com uses SSO.

Reply:
I mean that if you were from DOD, you'll also have an includedhealth.com e-mail, so the comment here isn't super accurate.

Reply:
I see what you mean. Dan Bolens has clarified for me that new engineers will all use @grandrounds.com until 1/15/2022 so I can simplify this.